src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -232,10 +233,11 @@ public void updateFilteredTaskList(Predicate<Task> predicate) {
     @Override
     public void updateSortedTaskList(Comparator<Task> comparator) {
         requireNonNull(comparator);
+        assert comparator instanceof DefaultComparator || comparator instanceof DeadlineComparator;
```

Comment:
Good that this is here...

Reply:
Thinking of making enums of these two comparators but I probably have no time to do that.